src/components/Header/Header.tsx

```diff
@@ -0,0 +1,156 @@
+import * as React from 'react';
+import { ColorPaletteProp } from '@mui/joy/styles';
+import Avatar from '@mui/joy/Avatar';
+import Badge from '@mui/joy/Badge';
+import Box from '@mui/joy/Box';
+import Button from '@mui/joy/Button';
+import Input from '@mui/joy/Input';
+import IconButton from '@mui/joy/IconButton';
+import ListDivider from '@mui/joy/ListDivider';
+import ListItemDecorator from '@mui/joy/ListItemDecorator';
+import Menu from '@mui/joy/Menu';
+import MenuItem from '@mui/joy/MenuItem';
+import Typography from '@mui/joy/Typography';
+import Sheet from '@mui/joy/Sheet';
+import Chip from '@mui/joy/Chip';
+import AddIcon from '@mui/icons-material/Add';
+import BubbleChartIcon from '@mui/icons-material/BubbleChart';
+import NotificationsIcon from '@mui/icons-material/Notifications';
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+
+export default function ColorInversionHeader() {
+  const [anchorEl, setAnchorEl] = React.useState<HTMLElement | null>(null);
+  const [color, setColor] = React.useState<ColorPaletteProp>('primary');
+  return (
+    <Sheet
+      variant="solid"
+      color={color}
+      invertedColors
+      sx={{
+        display: 'flex',
+        alignItems: 'center',
+        flexGrow: 1,
+        p: 2,
+        mx: -3,
+        my: -3,
+        borderRadius: { xs: 0, sm: 'xs' },
+        minWidth: 'min-content',
```

Comment:
same as every inline style

Reply:
I changed from mui/joy to mui library and concept. I have added the changes for approval.